prover/prover_dal/build.rs

```diff
@@ -0,0 +1,11 @@
+fn main() {
+    let database_url = std::env::var("DATABASE_PROVER_URL");
+    match database_url {
+        Ok(url) => {
+            println!("cargo:rustc-env=DATABASE_URL={}", url);
+        }
+        Err(_) => {
+            println!("cargo:warning=DATABASE_PROVER_URL is not set");
```

Comment:
Actually I'm not sure that we should do anything in the case DATABASE_URL is not present, we can simply do nothing in that case

Reply:
No, it won't. You can check one commit before, where it was supposed to just panic if there is no such var set, so that's why I have added this match. Packages will build normally if there is no DATABASE_URL set, but the build will break if we are trying to run core and prover altogether(I mean run here, but not just cargo build), so they will try to use the same DATABASE_URL. With adding this file we are simply saying - if you are trying to build prover_dal and there exists some DATABASE_URL that is okay to use(in our case it is DATABASE_PROVER_URL) - use it, but don't use a simple DATABASE_URL(which is a core DB URL)

Reply:
That said, there is no DATABASE_PROVER_URL env variable present while we are building images - so there is no need to try getting some, but we DO need to use it in CI where we are running init and similar things